lib/va_profile/military_personnel/service.rb

```diff
@@ -24,21 +24,20 @@ class Service < VAProfile::Service
       def get_service_history
         with_monitoring do
           edipi_present!
+
           response = perform(:post, identity_path, VAProfile::Models::ServiceHistory.in_json)
 
           ServiceHistoryResponse.from(@current_user, response)
         end
       rescue Common::Client::Errors::ClientError => e
-        if e.status == 404
+        error_status = e&.status
```

Comment:
You don't need the `&.` protected operator here, because the line above means `e` will always be defined here

Reply:
ah, yeah, good call.